bedrock/newsletter/templates/newsletter/existing.html

```diff
@@ -119,7 +120,7 @@ <h1>{{ _('Manage your Email Preferences') }}</h1>
           <tbody>
           {% for formpart in formset %}  {# each newsletter is a formset #}
             <tr class="subscription">
-              <th{% if formpart.initial['indented'] %} class="indented"{% endif %}>
```

Comment:
How will these rows be indented if we remove this? This value comes from basket and should result in the newsletter being visually indented.

Reply:
Ah ok, I wasn't sure what the indentation was for... it didn't look right and I thought it might be an artifact that wasn't needed. I'll put it back and see if I can make it look decent.

Reply:
Thanks. And yes, I appreciate the help :) It's meant to convey a grouping. There will be several under a main newsletter potentially.

Reply:
LCM can control ordering and indentation from basket admin, so they'll decide what is grouped under what.